<launch>
    <arg name="camera" default="/kinect_left/rgb" />
    <arg name="image_topic" default="$(arg camera)/image_raw" />
    <arg name="camera_info_topic" default="$(arg camera)/camera_info" />
    <arg name="marker_length" default="0.1" />
    <arg name="camera_frame_id" default="left_rgb_camera_link" />
    <node name="liveTags" pkg="aruco-tools" type="publishLiveTag.py" output="screen" args="$(arg image_topic) $(arg camera_info_topic) $(arg marker_length) $(arg camera_frame_id)"/>
</launch>